onnxruntime/test/providers/qnn/argmaxmin_op_test.cc

```diff
@@ -70,6 +70,7 @@ static void RunQDQArgMxxOpTest(const std::string& op_type, TestInputDef<float> i
 #else
   provider_options["backend_path"] = "libQnnHtp.so";
 #endif
+  provider_options["offload_graph_io_quantization"] = "0";
```

Comment:
Note: Most of our unit tests explicitly check that all nodes are assigned to QNN EP, so I disabled offloading for those. We do have pre-existing unit test that actually enables this option:
https://github.com/microsoft/onnxruntime/blob/5407c69028ae6dd4e87521aea147c22153d8e6c7/onnxruntime/test/providers/qnn/qnn_basic_test.cc#L1030